src/sentry/preprod/models.py

```diff
@@ -0,0 +1,222 @@
+from enum import IntEnum
+from typing import Any
+
+from django.db import models
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models.base import DefaultFieldsModel, region_silo_model
+from sentry.db.models.fields.bounded import (
+    BoundedBigIntegerField,
+    BoundedPositiveBigIntegerField,
+    BoundedPositiveIntegerField,
+)
+from sentry.db.models.fields.foreignkey import FlexibleForeignKey
+from sentry.db.models.fields.jsonfield import JSONField
```

Comment:
can you instead use the django JSONField -- I'm trying to factor this out

Reply:
sure, i'll create a new PR/migration for that.